crates/core/src/client/mod.rs

```diff
@@ -495,7 +495,7 @@ impl<P: Provider + Send + Sync> KakarotEthApi<P> for KakarotClient<P> {
         let newest_block = U256::from(newest_block);
         let oldest_block: U256 = if newest_block >= block_count { newest_block - block_count } else { U256::from(0) };
 
-        Ok(FeeHistory { base_fee_per_gas, gas_used_ratio, oldest_block, reward: None })
+        Ok(FeeHistory { base_fee_per_gas, gas_used_ratio, oldest_block, reward: Some(vec![vec![U256::ZERO]]) })
```

Comment:
necessary for `forge create` usage, otherwise...

🥁 🥁 🥁 🥁 

you guessed it! deserialization errors

(i wonder if theres a forked library we aren't including that would automatically handle all these weird edge cases for us--there has to be, right?)

Reply:
Why do we expect that None is a valid response?

Reply:
Because we don't have rewards on Starknet I think

Reply:
what about an empty `vec![]` I mean, it's not obvious that `None` could be a possible option

Reply:
Is it possible that returning `[[0]]` be interpreted as empty block?

Reply:
behold, the power of strong typing across multiple repos:

https://github.com/gakonst/ethers-rs/blob/master/ethers-core/src/types/fee.rs#L18

```
#[derive(Deserialize, Serialize, Debug, Clone)]
#[serde(rename_all = "camelCase")]
pub struct FeeHistory {
    pub base_fee_per_gas: Vec<U256>,
    pub gas_used_ratio: Vec<f64>,
    #[serde(deserialize_with = "deserialize_stringified_numeric")]
    /// oldestBlock is returned as an unsigned integer up to geth v1.10.6. From
    /// geth v1.10.7, this has been updated to return in the hex encoded form.
    /// The custom deserializer allows backward compatibility for those clients
    /// not running v1.10.7 yet.
    pub oldest_block: U256,
    /// An (optional) array of effective priority fee per gas data points from a single block. All
    /// zeroes are returned if the block is empty.
    #[serde(default)]
    pub reward: Vec<Vec<U256>>,
}

```

Reply:
lol, writing "optional" in docstring, but non making it `Option` in the code. Can we use an empty `vec<vec<>>` instead?

Reply:
trying rn, will see if it compiles